clients/geth/specular/Dockerfile

```diff
@@ -6,6 +6,8 @@ RUN apk add --no-cache make git
 COPY . /specular
 
 WORKDIR /specular/clients/geth/specular
+
+RUN apk add --no-cache python3 make g++
```

Comment:
why do we need this?

Reply:
On running docker I was getting missing python dependency.